src/main/java/org/folio/rest/impl/UsersAPI.java

```diff
@@ -593,7 +592,7 @@ private void getPG(PostgresClient postgresClient, User user, Handler<AsyncResult
    }else{
      Criterion c = new Criterion(
        new Criteria().addField(UserGroupAPI.ID_FIELD_NAME).setJSONB(false).
-       setOperation("=").setValue("'"+pgId+"'"));
+       setOperation("=").setVal(pgId));
```

Comment:
Can you replace the postgresClient.get by a PgUtil.getById invokation so that this duplicated code gets removed?